content/en/docs/concepts/instrumentation/zero-code.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Zero-Code
+description: >-
+  Learn how to add observability to an application without the need to touch the
+  code
+weight: 10
+aliases: [automatic]
+---
+
+As [ops](/docs/getting-started/ops/) you may want to add observability to an
+application (or to many applications!) without being able to add instrumentation
+to the code base. Or, you may be looking for a way to quickly gain some basic
+observability for an application, before diving into learning how to use the
+OpenTelemetry API & SDK for
+[code-based instrumentation](/docs/concepts/instrumentation/code-based).
```

Comment:
```suggestion
As [ops](/docs/getting-started/ops/) you might want to add observability to one
or more applications without having to edit the source, probably because you can't
or you're not allowed to. In other cases, you might be looking for a way to quickly
gain some OTel observability for a service before diving into learning how to use
the OpenTelemetry API & SDK for
[code-based instrumentation](/docs/concepts/instrumentation/code-based).
```